src/components/FormComponents/RelatedWorks.jsx

```diff
@@ -0,0 +1,266 @@
+                  fullWidth={false}
+                />
+              </Grid>
+              <Grid item xs>
+                <QuestionText>
+                  <I18n>
+                    <En>What is the relation type?</En>
+                    <Fr>
+                      Quel est le type de relation?</Fr>
+                  </I18n>
+                  <RequiredMark passes={dist.association_type} />
+                  <SupplementalText>
+                    <I18n>
+                      <En>
+                        <p>
+                          Specify the relationship from (A) the primary resource; to (B) the related resource. For example:
+                        </p>
+                        <ul>
+                          <li>Use 'Is New Version Of' to indicate the primary resource described in this metadata record (A) is a new version of (B) the related resource.</li>
+                          <li>Use 'Is Part of' to indicate the primary resource (A) is a subset of (B) the related larger resource.</li>
+                          <li>Use 'Has Part' to indicate the primary resource (A) is the larger work that includes (B) the related resource.</li>
+                          <li>Use 'Cites' to indicate that (A) cites (B).</li>
+                          <li>Use 'Is Cited by to indicate that (B) cites (A)</li>
+                        </ul>
+
+                      </En>
+                      <Fr>
+                        <p>
+                          Spécifiez la relation à partir de (A) la ressource principale ; à (B) la ressource associée. Par exemple:
+                        </p>
+                        <ul>
+                          <li>Utilisez "Est une nouvelle version de" pour indiquer que la ressource principale décrite dans cet enregistrement de métadonnées (A) est une nouvelle version de (B) la ressource associée.</li>
+                          <li>Utilisez "Fait partie de" pour indiquer que la ressource principale (A) est un sous-ensemble de (B) la ressource plus grande associée.</li>
+                          <li>Utilisez "A une partie" pour indiquer que la ressource principale (A) est le travail le plus important qui comprend (B) la ressource associée.</li>
+                          <li>Utilisez "Cites" pour indiquer que (A) cite (B).</li>
+                          <li>Utilisez "Est cité par" pour indiquer que (B) cite (A)</li>
+                        </ul>
+
+                      </Fr>
+                    </I18n>
+                  </SupplementalText>
+                </QuestionText>
+                <SelectInput
+                  value={dist.association_type}
+                  onChange={handleAssociationTypeChange()}
+                  options={Object.keys(associationTypeCode)}
+                  optionLabels={Object.values(associationTypeCode).map(
+                    ({ title }) => title[language]
+                  )}
+                  optionTooltips={Object.values(associationTypeCode).map(
```

Comment:
I feel like the current Tooltip that describe how the DataCite relation types map to ISO isn't particularly useful for the user. Could you update the tooltip so it explains the DataCite relation type using the definitions here?: https://datacite-metadata-schema.readthedocs.io/en/4.5/appendices/appendix-1/relationType/#issourceof